tests/Magento_Page_Loading.php

```diff
@@ -0,0 +1,473 @@
+<?php
+
+require_once 'vendor/autoload.php';
+
+class MagentoPageLoading extends Sauce\Sausage\WebDriverTestCase
+{
+    protected $base_url = 'http://127.0.0.1/';
+
+    public static $browsers = [
+        [
+            'browserName' => 'chrome',
+            'desiredCapabilities' => [
+                'version' => '45.0',
+                'platform' => 'OS X 10.10',
+            ],
+        ],
+    ];
+
+    const ADMIN_USERNAME    = 'denis';
+    const ADMIN_PASSWORD    = 'zavy123';
+    const ORDERS_COUNT      = 50;
+
+    const PAGE_TYPE_ADMIN       = 'admin';
+    const PAGE_TYPE_ORDER       = 'order';
+    const PAGE_TYPE_FRONT       = 'front';
+    const PAGE_TYPE_CUSTOMER    = 'customer';
+
+    private static $frontFailList       = [];
+    private static $customerFailList    = [];
+    private static $adminFailList       = [];
+    private static $orderFailList       = [];
+
+    /**
+     * Pages that don't use magento layout
+     *
+     * @var array
+     */
+    private $customLayoutPages = [
+        [
+            'link' => '/doneright/enhanced/dashboard',
```

Comment:
I moved these links into separate configuration file. We can use this file as a global configuration for testing module. But at the same time these links may seem irrelevant in the global configuration file. The data provider method is simple method to provide arguments for test method, so we should understand that the data provider and separate links in config file is a different things. We can use links from separate config file or data provider, that returns array with hard coded links for this test case, or data provider, that returns array with links from configuration, what do you say?